addons/mail/tests/test_mail_render.py

```diff
@@ -105,36 +77,36 @@ def setUpClass(cls):
         ]
 
         # link to mail template
-        cls.test_template_jinja = cls.env['mail.template'].create({
+        cls.test_template_inline_template = cls.env['mail.template'].create({
             'name': 'Test Template',
-            'subject': cls.base_jinja_bits[0],
-            'body_html': cls.base_jinja_bits[1],
+            'subject': cls.base_qweb_bits[0],
```

Comment:
This test makes no sense (and actually it should crash, isn't it ? shouldn't we enforce inline_template on non body fields ?)

Probably to propagate to other non body changes.